src/kernel.jl

```diff
@@ -390,6 +390,30 @@ function validate_gabor(σ::Real,λ::Real,γ::Real)
     end
 end
 
+"""
+    moffat(α, β, ls) -> k
+
+Constructs a 2D, symmetric Moffat kernel `k` with core width, `α`, and power, `β`.
+Size of kernel defaults to 4 * full-width-half-max or as specified in `ls`.
+See [this notebook](https://nbviewer.jupyter.org/github/ysbach/AO_2017/blob/master/04_Ground_Based_Concept.ipynb#1.2.-Moffat) for details.
+
+# Citation
+Moffat, A. F. J. "A theoretical investigation of focal stellar images in the photographic emulsion and application to photographic photometry." Astronomy and Astrophysics 3 (1969): 455.
+"""
+function moffat(α::Real, β::Real, ls::Tuple{Integer, Integer})
+    ws = map(n->(ceil(Int,n)>>1), ls)
+    R = CartesianIndices(map(w->IdentityUnitRange(-w:w), ws))
+    α2 = α^2
+    amp = (β - 1)/(π * α2)
+    function df(I::CartesianIndex)
+        x = SVector(Tuple(I))
+        sum(x.^2)
+    end
```

Comment:
you could move this part outside this function and add an `@inline` notation: in my local machine I observe a boost from 49.346 μs to 2 μs

Reply:
Thanks, implemented in https://github.com/JuliaImages/ImageFiltering.jl/pull/147/commits/36c0a8a6fad366a65b23d2a2d3d73c3443f1ebdd